test/onnx/test_operators.py

```diff
@@ -1013,6 +1020,29 @@ def forward(self, x, y):
 
         unregister_custom_op_symbolic('::embedding', _onnx_opset_version)
 
+    def test_shape_value_map(self):
+        class rSoftMax(torch.nn.Module):
+            def __init__(self, radix, cardinality):
+                super().__init__()
+                self.radix = radix
+                self.cardinality = cardinality
+
+            def forward(self, x):
+                batch = x.size(0)
```

Comment:
Please add a comment explaining how this code relates to the functionality being tested.